st2common/tests/unit/services/test_action.py

```diff
@@ -115,6 +118,16 @@ def test_request_invalid_parameters(self):
         liveaction = LiveActionDB(action=ACTION_REF, parameters=parameters)
         self.assertRaises(jsonschema.ValidationError, action_service.request, liveaction)
 
+    def test_request_optional_parameter_none_value(self):
+        parameters = {'hosts': 'localhost', 'cmd': 'uname -a', 'a': None}
+        request = LiveActionDB(action=ACTION_REF, parameters=parameters)
+        request, _ = action_service.request(request)
+
+    def test_request_optional_parameter_none_value_no_default(self):
+        parameters = {'hosts': 'localhost', 'cmd': 'uname -a', 'b': None}
+        request = LiveActionDB(action=ACTION_REF, parameters=parameters)
+        request, _ = action_service.request(request)
```

Comment:
This doesn't seem to test much. Are you just validating that request doesn't throw? Same question about prev test case.